frontend/src/scenes/trends/PersonsModal.tsx

```diff
@@ -167,144 +168,146 @@ export function PersonsModal({
                     )
                 }
                 width={600}
-                className="person-modal"
             >
-                {isInitialLoad ? (
-                    <div className="p-4">
-                        <Skeleton active />
-                    </div>
-                ) : (
-                    people && (
-                        <>
-                            <div className="p-4">
-                                <LemonInput
-                                    icon={<IconMagnifier />}
-                                    allowClear
-                                    placeholder="Search for persons by email, name, or ID"
-                                    onChange={(value) => {
-                                        setSearchTerm(value)
-                                        if (!value) {
-                                            setFirstLoadedActors(firstLoadedPeople)
-                                        }
-                                        setPersonsModalFilters(value, people, filters)
-                                    }}
-                                    value={searchTerm}
-                                />
-                            </div>
-                            {!!people.crossDataset?.length && people.seriesId !== undefined && (
-                                <div className="data-point-selector">
-                                    <Select value={people.seriesId} onChange={(_id) => switchToDataPoint(_id)}>
-                                        {people.crossDataset.map((dataPoint) => (
-                                            <Select.Option
-                                                value={dataPoint.id}
-                                                key={`${dataPoint.action?.id}${dataPoint.breakdown_value}`}
-                                            >
-                                                <InsightLabel
-                                                    seriesColor={getSeriesColor(dataPoint.id)}
-                                                    action={dataPoint.action}
-                                                    breakdownValue={
-                                                        dataPoint.breakdown_value === ''
-                                                            ? 'None'
-                                                            : dataPoint.breakdown_value?.toString()
-                                                    }
-                                                    showCountedByTag={showCountedByTag}
-                                                    hasMultipleSeries={hasMultipleSeries}
-                                                />
-                                            </Select.Option>
-                                        ))}
-                                    </Select>
+                <div className="persons-modal space-y-2">
```

Comment:
Side question: is it intentional that the list has a padding now?

| Before | After |
| --- | --- |
| <img width="623" alt="Screen Shot 2022-08-09 at 12 34 45" src="https://user-images.githubusercontent.com/4550621/183628309-30befa44-5b72-4253-9651-dea730314d44.png"> | <img width="630" alt="Screen Shot 2022-08-09 at 12 34 16" src="https://user-images.githubusercontent.com/4550621/183628317-6c0c208e-9129-45a5-93db-2a53a833ca33.png"> |

Reply:
Not intentional perse but intentionally not fixing it as this component deviates from the standard Modal styles anyways. This modal is incredibly buggy anyway so I am tempted to leave it this way and revisit with purpose.